### ISSUE TITLE
Tidal plugin only displays self-created playlists, not subscribed ones

### ISSUE THREAD

**Issue by mika-fischer** [LMS-Community/slimserver#840]
Compare the Tidal web UI:
<img width="382" alt="2022-12-04_19h43_08" src="https://user-images.githubusercontent.com/426158/205509356-28ff1872-b1b9-42c3-9d41-b2deb1f33631.png">

with the Squeezebox web UI:
<img width="183" alt="2022-12-04_19h43_35" src="https://user-images.githubusercontent.com/426158/205509382-06548fe9-327d-45cc-8acc-2e2a99478c1a.png">

Is there anything I can do to fix this?

**Comment by michaelherger** [comment]
Is this a recent change? There indeed are two different ways to get user playlists. I'm wondering whether the one we're using was changed to be really only the playlists created by the user. It'll probably be easy to change on the server side.

**Comment by mika-fischer** [comment]
Sorry, I don't know as I created my account only a couple of days ago.

**Comment by michaelherger** [comment]
Can you try again?

**Comment by mika-fischer** [comment]
Now I get *only* the subscribed playlists but not my own 😄 
<img width="189" alt="2022-12-05_08h24_17" src="https://user-images.githubusercontent.com/426158/205577421-76457613-28a2-49dc-98bc-69d56b525d79.png">

**Comment by michaelherger** [comment]
Hmm... I tested with two accounts, and they work as expected... Please let me know should the situation not improve. I could imagine there's some caching involved. Did you create your own playlist before or after you added the public one?

**Comment by mika-fischer** [comment]
> I could imagine there's some caching involved.

Yes, it seems so. I just created another playlist and it appears correctly.

> Did you create your own playlist before or after you added the public one?

Not 100% sure about the order, but I think I created my own playlist after subscribing to the public one...

Restarting the logitechmediaserver service also didn't help.

It's no big deal for me, I can just recreate this missing playlist. But maybe for other users it could be an issue...

If I can test anything, let me know. And thanks a lot for the very quick fix!
